test/integration/idle_timeout_integration_test.cc

```diff
@@ -164,10 +178,12 @@ TEST_P(IdleTimeoutIntegrationTest, PerStreamIdleTimeoutAfterBidiData) {
   auto response = setupPerStreamIdleTimeoutTest();
 
   sleep();
-  upstream_request_->encode100ContinueHeaders(Http::TestHeaderMapImpl{{":status", "100"}});
+  upstream_request_->encode100ContinueHeaders(
+      Http::TestHeaderMapImpl{{":status", std::to_string(enumToInt(Http::Code::Continue))}});
```

Comment:
Can you explain why you opted to change all of these parameters? To me, this makes the test harder to understand on first glance.

Reply:
This is just to make all the tests in this file use an `enum` rather than string error codes as per @jmarantz's suggestion.